test/test-deploy.sh

```diff
@@ -0,0 +1,20 @@
+# Makes the ubi8-s2i-nodejs image.
+make all
+
+# Login into openshift docker registry.
+docker login -u `oc whoami` -p `oc whoami -t` 172.30.1.1:5000
+
+# Creates a new project.
+oc new-project nodeshift
+
+# Creates a tag on openshift docker registry based on the local created image.
+docker tag nodeshift/ubi8-s2i-nodejs:12.x 172.30.1.1:5000/nodeshift/ubi8-s2i-nodejs:12.x
+
+# Pushes the image to the openshift docker registry.
+docker push 172.30.1.1:5000/nodeshift/ubi8-s2i-nodejs:12.x
+
+# Creates a new app based on the pushed image.
+oc new-app 172.30.1.1:5000/nodeshift/ubi8-s2i-nodejs:12.x~https://github.com/nodeshift/nodejs-rest-http
```

Comment:
Is this `172.x.x.x.` IP address always going to be correct?

Reply:
I've have no idea if will always be correct. This is just what currently exists  in nodeshift/centos-s2i-nodejs.